### ISSUE TITLE
Document how to override paparazzi system properties

### ISSUE THREAD

**Issue by AlexanderGH** [cashapp/paparazzi#1335]
Related to https://github.com/cashapp/paparazzi/issues/1334, I wanted to move the snapshots out of src/test and into the root of the module, however, it's not clear what the _recommended_ way to set [system properties to override the default paparazzi options](https://github.com/cashapp/paparazzi/blob/ef18bed4e4201c5a11ed6603d16228d18a840e2c/paparazzi/build.gradle#L139) is. It would be nice if the paparazzi docs could be updated with the recommended pattern for Gradle integrations.

**Comment by jrodbx** [comment]
Setting jvm system properties is neither unique to Paparazzi nor Gradle; this is by design in case someone decided to contribute a bazel plugin in the future.

The following should work, in your `build.gradle`:
```
tasks.withType(Test).configureEach {
  systemProperty(
    "paparazzi.snapshot.dir",
    YOUR_CUSTOM_SNAPSHOT_DIR
  )
}
```

I'm reluctant to call this the "recommended" way, since moving things into the root module might break Gradle's overlapping outputs even more (since all modules will share a single rather than separate output directories), but that should unblock what you're looking to accomplish.

**Comment by AlexanderGH** [comment]
Thanks. Yep, when i said root, i meant root of each module not the entire gradle build project. But at this point I'll likely just end up waiting for a formal fix for #1334 so it's a moot point in my case.
